### DISCUSSION TITLE
change entity size [solved]

### DISCUSSION THREAD

**Discussion by HttpPython** [Minestom/Minestom#1629] [Help]
hey i want to change the size of a slime to -1 how can i do that?

**Comment by zax71** [comment]
https://javadoc.minestom.net/net/minestom/server/entity/metadata/other/SlimeMeta.html#setSize(int) Just use the JavaDocs

**Reply by HttpPython** [reply]
InstanceContainer instanceContainer = Lobby.getInstance().getWorldManager().getInstance("lobby");
        LivingEntity entity = new LivingEntity(EntityType.SLIME);
        entity.setCustomNameVisible(true);
        entity.setCustomName(Component.text("test"));
        entity.setInvisible(true);
        SlimeMeta slimeMeta = (SlimeMeta) entity.getEntityMeta();
        slimeMeta.setSize(-1);
        entity.setInstance(instanceContainer);
        player.addPassenger(entity);
        
        is this right?

**Reply by HttpPython** [reply]
well now the slime is in my body

**Reply by zax71** [reply]
Would you be able to set this as the accepted answer if it worked :)
